### PR TITLE
[ui only] Mobile network bottom sheet

### PR DESCRIPTION
**Pull request by rasom**
part of #7288 

In order to test it turn on/off wifi  on the device. Before merging bottom sheet will be hidden, currently that's just UI.

### CODE REVIEW

**Comment by ghost** [comment]
### Pull Request Checklist
- [ ] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status.html))?

**Comment by rasom** [comment]
@andmironov so it looks like "Continue syncing" and "Stop syncing" rows are components, right?

**Review by dmitryn — APPROVED** [review]

**Review by mandrigin — APPROVED** [review]

File: src/status_im/ui/components/bottom_sheet/subs.cljs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+(ns status-im.ui.components.bottom-sheet.subs
+  (:require [re-frame.core :as re-frame]))
+
+(re-frame/reg-sub
+ :bottom-sheet
+ (fn [{:bottom-sheet/keys [show? view]}]
+   {:show? show?
```
Comment (vitvly) [inline src/status_im/ui/components/bottom_sheet/subs.cljs:7]: Minor: there was a tiny macro in open-bounty repo (https://github.com/status-im/open-bounty/blob/31c85bfd94ab01b47e6cdbdff79982c2f5bd65ad/src/clj/commiteth/util/util.clj#L19) for constructing these maps.

File: src/status_im/ui/components/checkbox/view.cljs [hunk 2]
```diff
@@ -3,21 +3,28 @@
...
                          :accessibility-label accessibility-label}]]
-      [react/touchable-highlight (merge {:style               (merge styles/wrapper style)
+      [react/touchable-highlight (merge {:style               (merge
+                                                               (icon-check-container checked?)
+                                                               styles/wrapper
+                                                               style)
                                          :accessibility-label accessibility-label}
                                         (when on-value-change {:on-press #(on-value-change (not checked?))}))
-       [react/view (icon-check-container checked?)
+       [react/view {}
```
Comment (vitvly) [inline src/status_im/ui/components/checkbox/view.cljs:27]: Probably map arg is not necessary if empty.

**Review by vitvly — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/components/bottom_sheet/subs.cljs (hunk 1 above)
Comment (yenda) [reply src/status_im/ui/components/bottom_sheet/subs.cljs:7]: or select-keys

Comment (rasom) [reply src/status_im/ui/components/bottom_sheet/subs.cljs:7]: I'm not sure that I understand how `select-keys` will solve this. Don't want to introduce new a new macro here.

**Pull request merged**